Userland/Utilities/ls.cpp

```diff
@@ -142,6 +143,22 @@ int main(int argc, char** argv)
     for (size_t i = 0; i < paths.size(); i++) {
         auto path = paths[i];
 
+        if (flag_recursive && Core::File::is_directory(path)) {
+            size_t subdirs = 0;
+            Core::DirIterator di(path, Core::DirIterator::SkipParentAndBaseDir);
+
+            if (di.has_error()) {
+                status = 1;
+                fprintf(stderr, "%s: %s\n", path, di.error_string());
+            }
+
+            while (di.has_next()) {
+                const String directory = di.next_full_path();
```

Comment:
`const` is pretty pointless here, but if you really want to keep it there, please switch the file to use east-const (to comply with the project coding style) and also use it here.
Note that if you use `Vector<String> paths`, making this not `const` and later moving it into that vector is preferred.

Reply:
I simply removed the `const` here. I am also using `move()` on the `directory` now as per your recommendation.